gefserver/pier/pier.go

```diff
@@ -45,6 +45,7 @@ type dockerConnection struct {
 	stageIn        internalImage
 	fileList       internalImage
 	copyFromVolume internalImage
+	mavenEGI	   internalImage
```

Comment:
Formatting is wrong. There is "go fmt ./..." command that will take care of tabs and spaces